pkg/map/iface.go

```diff
@@ -30,3 +25,22 @@ type Route struct {
 	// ProxyAuth represents optional auth for kedge.
 	ProxyAuth tokenauth.Source
 }
+
+// ErrorKedgeDestination displays that host/port is not a kedge destination. We are not using errors.New, because sometimes we need
+// to check for type of the error (for example in winch/grpc/proxy.go).
+type ErrorNotKedgeDestination struct {
+	host string
+	port string
+}
+
+func NotKedgeDestinationErr(host string, port string) error {
+	return &ErrorNotKedgeDestination{host: host, port: port}
+}
+
+func (e *ErrorNotKedgeDestination) Error() string {
```

Comment:
`(e ErrorNotKedgeDestination)` instead -> you don't need pointer for that. (BTW you instantiate as Value anyway)